tests/qaautoRegistration.spec.js

```diff
@@ -0,0 +1,127 @@
+    await expect(page.locator(invalidNameMsg)).toHaveText('Name is invalid');
+    await page.locator(signUpName).fill(wrongLengthUserDataMin.username);
+    await expect(page.locator(invalidNameMsg)).toHaveText('Name has to be from 2 to 20 characters long');
+    await page.locator(signUpName).fill(wrongLengthUserDataMax.username);
+    await expect(page.locator(invalidNameMsg)).toHaveText('Name has to be from 2 to 20 characters long');
+  });
+
+
+ test('Verify Last Name', async ({ page }) => {
+    await page.locator(signUpLastName).focus();
+    await page.locator(signUpEmail).focus();
+    await expect(page.locator(lastnameRequired)).toHaveText('Last name required');
+    await expect(page.locator(signUpLastName)).toHaveCSS('border-color', 'rgb(220, 53, 69)');
+    await page.locator(signUpLastName).fill(wrongUserData.lastname);
+    await expect(page.locator(invalidLastNameMsg)).toHaveText('Last name is invalid');
+    await page.locator(signUpLastName).fill(wrongLengthUserDataMin.lastname);
+    await expect(page.locator(invalidLastNameMsg)).toHaveText('Last name has to be from 2 to 20 characters long');
+    await page.locator(signUpLastName).fill(wrongLengthUserDataMax.lastname);
+    await expect(page.locator(invalidLastNameMsg)).toHaveText('Last name has to be from 2 to 20 characters long');
+  });
+
+  test('Verify Email validation', async ({ page }) => {
+    await page.locator(signUpEmail).focus();
+    await page.locator(signUpEmail).fill('');
+    await page.locator(signUpPwd).focus();
+    await expect(page.locator(invalidEmailMsg)).toHaveText('Email required');
+    await expect(page.locator(signUpEmail)).toHaveCSS('border-color', 'rgb(220, 53, 69)');
+    await page.locator(signUpEmail).fill(wrongUserData.email);
+    await expect(page.locator(invalidEmailMsg)).toHaveText('Email is incorrect');
+
+  });
+
+  test('Verify PWD validation', async ({ page }) => {
+    await page.locator(signUpPwd).focus();
+    await page.locator(signUpPwd).fill('');
+    await page.locator(signUpEmail).focus();
+    await expect(page.locator(invalidPwdMsg)).toHaveText('Password required');
+    await expect(page.locator(signUpPwd)).toHaveCSS('border-color', 'rgb(220, 53, 69)');
+    await page.locator(signUpPwd).fill(wrongUserData.password);
+    await expect(page.locator(invalidPwdMsg)).toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter');
+    await expect(page.locator(registerBtn)).toBeDisabled();
+
+  });
+   
+  test('Verify PWD validation', async ({ page }) => {  
+    await page.locator(signUpRepeatPwd).focus();
+    await page.locator(signUpRepeatPwd).fill('');
+    await page.locator(signUpEmail).focus();
+    await expect(page.locator(invalidRptPwd)).toHaveText('Re-enter password required');
+    await page.locator(signUpPwd).fill(validUserData.password);
+    await page.locator(signUpRepeatPwd).fill(wrongUserData.rptPwd);
```

Comment:
you entered values but nothing happens after

did you forget to make  an assertion?